source/stable-release_iot/user-guide-a12.rst

```diff
@@ -0,0 +1,418 @@
+`<https://docs.01.org/celadon/getting-started/build-source.html#set-up-the-development-environment>`_ for |C| build host setup.
+
+Users of Celadon-in-VM (CIV) release must ensure that Celadon platform host OS hardening measures are in place to ensure that the host OS could be treated as part of the secure computing base. This is essential to ensuring CIV security could be trusted in CIV operations.
+
+.. code-block:: bash
+
+	# Copy the artifact
+	$ cp caas-releasefiles-userdebug.tar.gz <working_dir>
+
+	# Extract files
+	$ cd <working_dir>
+	$ tar xzvf caas-releasefiles-userdebug.tar.gz 
+	
+	# Prepare setup_host.sh
+	$ chmod +x ./scripts/setup_host.sh
+	# Update the host
+	# If prompted, answer ‘Y’ for all options to go ahead with changes
+	$ sudo -E ./scripts/setup_host.sh 
+
+Create Android VM Image:
+
+Create Android CIV image for running as VM in ADL target
+
+.. code-block:: bash
+
+	# Change directory
+	$ cd ~
+	# Generate CIV disk image from caas-flashfiles.
+	# The script and flashfiles have already been extracted from caas-releasefiles-userdebug.tar.gz
+	# Wait for "Flashing is completed" msg from script.
+	$ sudo -E ./scripts/start_flash_usb.sh caas-flashfiles-xxxxx.zip  (Use --display-off if flashing remotely)
+
+Running Android* 12 
+********************
+
+This section describes the steps to run Android 12, Yocto, Windows 10 and Ubuntu Guest VMs on the ADL-N platform
+
+* VM Launch
+Launch Celadon Android Guest VM
+
+.. code-block:: bash
+
+	# Launch the Android CIV Guest VM
+	$ cd ~
+	$ sudo vm-manager -b civ-sriov
+	
+Guest VM Configuration Options
+
+Change Guest VM Memory and Number of CPUs:
+For Android 12 Guest VM only, edit the memory and vcpu sections of the configuration ini file at <workspace>/.intel/.civ/civ-sriov.ini.
+
+.. code-block:: bash
+
+	# Enable USB Devices in Guest VM
+	# [memory]
+	# size=4G
+
+	# [vcpu]
+	# num=4G
+
+.. note::
+	A passthrough device option can only be used once, because a device can be passthrough to only 1 guest VM at a time
+	For Android 12 guest VM, the passthrough is defined in the configuration ini file.
+
+Android 12 guest VM USB device passthrough:
+
+This section describes the steps to run Android 12, ADL platform.
+
+.. code-block:: bash
+
+	# Find the PCI ID of the USB device
+	$ $ lspci -nn -D | grep USB
+		0000:00:14.0 USB controller [0c03]: Intel Corporation Device [8086:7ae0] (rev 11)
+		0000:00:14.1 USB controller [0c03]: Intel Corporation Device [8086:7ae1] (rev 11)
+		0000:05:00.0 USB controller [0c03]: Intel Corporation Thunderbolt 4 NHI [Maple Ridge 4C 2020] [8086:1137]
+		0000:07:00.0 USB controller [0c03]: Intel Corporation Thunderbolt 4 USB Controller [Maple Ridge 4C 2020] [8086:1138]
+
+	# Edit the passthrough section of the configuration ini file at <workspace>/.intel/.civ
+	#[passthrough]
+	#specified the PCI id here if you want to passthrough it to guest, separate them with comma
+	$ passthrough_pci=0000:00:14.0,0000:00:14.1,0000:05:00.0,0000:07:00.0,
```

Comment:
This looks wrong. It should be:

> [passthrough]
#specified the PCI id here if you want to passthrough it to guest, separate them with comma
passthrough_pci=0000:00:14.0,0000:00:14.1,0000:05:00.0,0000:07:00.0,